pycloudlib/oci/cloud.py

```diff
@@ -274,7 +275,10 @@ def launch(
                 https://docs.cloud.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm
             user_data: used by Cloud-Init to run custom scripts or
                 provide custom Cloud-Init configuration
+            subnet_id: string, OCID of subnet to use for instance.
```

Comment:
nit: doc string should be part of commit that added argument
non-nit: doc string should describe which arguments take precedence over others (subnet_id > subnet_name)

Reply:
ack, both have been addressed.